src/test/java/Homework18.java

```diff
@@ -0,0 +1,42 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework18 extends BaseTest{
+    @Test()
```

Comment:
Hi,
I have checked your homework 18, but there are a lot of errors in your code that make the test script unable to execute. Please check it carefully again and correct it.
Best wishes from the HW team (Khanh, khanh@testpro.io)

Here are some of my suggestions:
1. You need to use correct annotations @Test testng, @Test not is @Test()